src/main/java/com/backbase/ct/bbfuel/setup/CapabilitiesDataSetup.java

```diff
@@ -179,12 +188,34 @@ private void ingestActionsPerUser() {
 
     private void ingestBillPayUsers() {
         if (this.globalProperties.getBoolean(PROPERTY_INGEST_BILLPAY)) {
-            this.accessControlSetup.getLegalEntitiesWithUsersExcludingSupport()
-                .stream()
-                .map(LegalEntityWithUsers::getUserExternalIds)
-                .flatMap(List::stream)
-                .collect(Collectors.toList())
-                .forEach(this.billpayConfigurator::ingestBillPayUser);
+            this.accessControlSetup.getLegalEntitiesWithUsersExcludingSupport().forEach((le) -> {
+
+                List<Account> accounts = new ArrayList<>();
+                String externalUserId = le.getUserExternalIds().get(0);
+                loginRestClient.login(externalUserId, externalUserId);
+                String externalLegalEntityId = this.userPresentationRestClient.
```

Comment:
Not related to your PR but we could remove this bypass if we have the `externalLegalEntityId ` already during loading. Now It's either defined in the input json file or generated during ingestion.

Reply:
Currently the legal-entities-with-users json file, not all legal entities in the file have an id
for example the first object only contains the list of users: 
`{
    "users": [
      {
        "externalId": "user"
      }
    ]
  },`

Reply:
yes, we can optimize this in the future